### ISSUE TITLE
The "path" argument must be of type string. Received type undefined

### ISSUE THREAD

**Issue by liyasthomas** [stylelint/stylelint#4812]
Nuxt project produce the following error on development mode.

![image](https://user-images.githubusercontent.com/10395817/83268959-ffd4b680-a1e3-11ea-814a-c24cd40ec11d.png)

@nuxtjs/stylelint-module: `v4.0.0`

> Which rule, if any, is the bug related to?

default/no rules needed

> What code is needed to reproduce the bug?

can't detect from where the error originated.

> What stylelint configuration is needed to reproduce the bug?

default config

> Which version of stylelint are you using?

`v13.5.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

"*.{scss,css,vue}": "stylelint"

`lint-staged` on pre-commit along with nuxt-stylelint-module

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

not sure

> What did you expect to happen?

No stylelint error

> What actually happened (e.g. what warnings or errors did you get)?

e.g. "The following warnings were flagged:"

Nuxt project produce the following error on development mode.
```
ERROR StylelintError

The "path" argument must be of type string. Received type undefined
```

**Comment by jeddy3** [comment]
@liyasthomas Thanks for the report and for using the template.

I suggest raising an issue in the [stylelint-module tracker](https://github.com/nuxt-community/stylelint-module#readme) as this appears to be related to that extension, rather than stylelint itself.

I'll close the issue here. However, you're welcome to reopen the issue if you can reproduce the issue using the stylelint CLI.
